framework/include/meshgenerators/PatternedMeshGenerator.h

```diff
@@ -47,6 +47,11 @@ class PatternedMeshGenerator : public MeshGenerator
   /// Holds a mesh for each row, these will be stitched together in the end
   std::vector<std::unique_ptr<ReplicatedMesh>> _row_meshes;
 
+  /// Merges the subdomain name maps between two meshes
```

Comment:
Add docstring to explain what each map contains. 

Add a note about the conflict issue